src/java/org/apache/cassandra/cql3/functions/MathFcts.java

```diff
@@ -0,0 +1,140 @@
+ /*
+  * Licensed to the Apache Software Foundation (ASF) under one
+  * or more contributor license agreements.  See the NOTICE file
+  * distributed with this work for additional information
+  * regarding copyright ownership.  The ASF licenses this file
+  * to you under the Apache License, Version 2.0 (the
+  * "License"); you may not use this file except in compliance
+  * with the License.  You may obtain a copy of the License at
+  *
+  *     http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+ package org.apache.cassandra.cql3.functions;
+
+ import java.nio.ByteBuffer;
+ import java.util.Collection;
+ import java.util.List;
+ import java.util.stream.Collectors;
+
+ import com.google.common.collect.ImmutableList;
+ import org.slf4j.Logger;
+ import org.slf4j.LoggerFactory;
+
+ import org.apache.cassandra.db.marshal.*;
+ import org.apache.cassandra.transport.ProtocolVersion;
+
+ public class MathFcts
```

Comment:
Nits: As the class contains only static methods it should be final and the constructor should be private.

Reply:
Good point, I think we should probably also name it something like `MathFcts**Util**` similar to others like `SyncUtil`, etc

Reply:
Ok, I made it final, added a private constructor, and renamed it to `MathFctsUtil`, I also changed the test class name to match.